recipes/django-elasticsearch-dsl/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "django-elasticsearch-dsl" %}
+{% set version = "7.2.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/django-elasticsearch-dsl-{{ version }}.tar.gz
+  sha256: f9163ffe314a0562f00789d34cc923332e1d51236bf7d5a38bfa8ea72399ceac
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - elasticsearch-dsl >=7.2.0
+    - django >=1.9.6
+    - python >=3.6
+    - six
+
+test:
+  imports:
+    - django_elasticsearch_dsl
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/sabricot/django-elasticsearch-dsl
```

Comment:
This actually resolves now to https://github.com/django-es/django-elasticsearch-dsl  Can you update? I was really confused for a minute.